README.md

```diff
@@ -37,6 +37,7 @@ Inputs are provided using the `with:` section of your workflow YML file.
 | fleet | The slug of the fleet (eg: `my_org/sample_fleet`) for which the release is for | true | |
 | environment | Domain of API hosting your fleets | false | balena-cloud.com |
 | versionbot | Tells action to use Versionbot branch for versioning | false | false |
+| create_ref | Attempt to tag the git commit with the release version | false | false |
```

Comment:
```suggestion
| create_ref | Tag the git commit with the release version | false | false |
```

Attempt sounds like it might not tag and complete still.

What do you think about changing the name to just `tag` ?

Reply:
I said "attempt" because we want to ignore failures if the tag already exists. Like pushing to master where versionbot has already created a tag, and the balena dashboard version might be at rev0 (rev0 is not appended) so they would conflict.

That said, we don't need to advertise that we are ignoring certain errors. I agree we could just change this to your suggestion.

Reply:
I think `tag` is already an overloaded word in a lot of contexts, but ultimately it's up to you?

Reply:
`create_ref` seems good to me then if the upstream function is called that!

Let's just change the description to not include "Attempt". My thought process is, for debugging I want the action to always do what it's told and error if the state hasn't changed given the instructions provided (if change was expected).

Reply:
Feel free to self-certify after reading my comment^